hedera-node/hedera-token-service-impl/src/test/java/com/hedera/node/app/service/token/impl/test/handlers/transfer/CustomFeeAssessmentStepTest.java

```diff
@@ -295,6 +295,24 @@ void hbarFixedFeeAndRoyaltyFeeWithFallbackNetOfTransfers() {
         assertThatTransfersContains(givenOp.transfers().accountAmounts(), expectedGivenOpHbarTransfers);
     }
 
+    //    @Test
```

Comment:
Can this be removed?

Reply:
Ah yes, removed. Thanks @thomas-swirlds-labs . Accidentally got committed.